bcl2fastq/index_summary.py

```diff
@@ -152,12 +191,23 @@ def update_known_index_bar(run_json):
     [dep.Input('pruned_unknown_hidden', 'children')]
 )
 def update_unknown_index_pie(run_json):
+    """ When input (sample_run_hidden) is changed, function is rerun to get update bar graph unknown indices
+
+               Attributes:
+                   pruned(list): select run_alias'
+                   data(list): empty list appended with parameters of "For" function
+               Parameters:
+                   run_json(list): json format of run variable
+
+               Returns: stacked bar graph of descending 30 unknown indices
```

Comment:
It looks to me like this returns data and layout information, but I think it's the a separate method (part of the dashi library) that converts that data into a stacked bar graph.